src/main/java/edu/ucsb/cs156/gauchoride/repositories/RideRepository.java

```diff
@@ -0,0 +1,12 @@
+package edu.ucsb.cs156.gauchoride.repositories;
+
+import edu.ucsb.cs156.gauchoride.entities.Ride;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+
+@Repository
+public interface RideRepository extends CrudRepository<Ride, Long> {
+  Iterable<Ride> findAllByStudent(String student);
```

Comment:
I would strongly encourage you to replace String student with Long user_id, and store here (and look up by) the rider's userid.